### DISCUSSION TITLE
Linkwarden 2.6.0 not generating "preserved formats"

### DISCUSSION THREAD

**Discussion by bestplay9384** [tteck/Proxmox#3474] [Help Needed] [answered]
I found out that linkwarden freshly installed DO NOT generate any "preserved formats" (screenshots, PDFs, readable).
As far as i found out what the issues is, it turned out that some dependencies are missing, and they are not mentioned in linkwarden documentation neither :/

However by doing this: https://github.com/linkwarden/linkwarden/issues/655#issuecomment-2241610045 i managed to make it work. 
As the discussion in URL shows, issue is there since 2.6.0, and it is for now the newest version.
Please consider adding this to install script :)

Thanks

**Comment by tteck** [comment, marked as answer]
Thanks for the heads up! https://github.com/tteck/Proxmox/commit/4a1e02a9abef75cd2cd672620c5e792bb8f11332 https://github.com/tteck/Proxmox/commit/b66590f976ad2824a4d4474b332ac6bc7a66cec8

**Reply by the-dunce** [reply]
I ran the script earlier today and had this same issue, but I just tried again and it's working fine after the update. Thanks @tteck, you are a legend!
